### PR TITLE
Add composer scripts

### PR DESCRIPTION
**Pull request by GromNaN**
Ease dev process by adding shortcuts to the common commands.

Also adding me to the authors of the library, not because I've deleted almost as many lines as I've added, but because it's become my full-time activity.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ Composer.
 
 The test suite may be executed with:
 
-```
-$ vendor/bin/simple-phpunit
+```console
```
Comment (alcaeus) [inline CONTRIBUTING.md:29]: `console` does not exist - the "language" you're looking for is [`Shell`](https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/lib/linguist/languages.yml#L6393)

**Review by alcaeus — COMMENTED** [review]
Highlighting for shell scripts needs to be fixed - the rest LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (GromNaN) [reply CONTRIBUTING.md:29]: It's an [alias](https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/lib/linguist/languages.yml#L6482).

Comment (alcaeus) [reply CONTRIBUTING.md:29]: Ah, good one!

**Review by alcaeus — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ By default, the `simple-phpunit` binary chooses the correct PHPUnit version for
 the PHP version you are running. To run tests against a specific PHPUnit
 version, use the `SYMFONY_PHPUNIT_VERSION` environment variable:
 
-```
+```console
 $ SYMFONY_PHPUNIT_VERSION=7.5 vendor/bin/simple-phpunit
```
Comment (jmikola) [inline CONTRIBUTING.md:51]: I realize this is outside the PR scope, but it's a good opportunity to bump this to a later version. Given https://phpunit.de/supported-versions.html, I don't think anything older than 8.5 would be used in practice.

Alternatively, we can remove this example and just link to https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script for canonical documentation.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -108,55 +108,63 @@ The following environment variables are used for [CSFLE testing](https://github.
...
 
-## Automatic code refactoring
+### Refactoring
 
 The library uses [rector](https://getrector.com/) to refactor the code for new features.
 To run automatic refactoring, use the `rector` command:
 
-```
+```console
 $ vendor/bin/rector
```
Comment (jmikola) [inline CONTRIBUTING.md:168]: Did you mean to change this to the `fix:` or `check:` command? Or are you intentionally calling the original binary here?

**Review by jmikola — APPROVED** [review]
Small feedback but LGTM regardless.

File: CONTRIBUTING.md (hunk 3 above)
Comment (GromNaN) [reply CONTRIBUTING.md:168]: To be honest, I hesitated. In the end, I think you should offer the 2 entry points into the document: Composer scripts in the header and the path to the binaries in detail.

**Pull request merged**